substrate/frame/support/procedural/src/construct_runtime/expand/call.rs

```diff
@@ -207,5 +219,61 @@ pub fn expand_outer_dispatch(
 				}
 			}
 		)*
+
+		/// SkipCheckIfFeeless
+		#[derive(
+			#scrate::RuntimeDebugNoBound,
+			#scrate::CloneNoBound,
+			#scrate::EqNoBound,
+			#scrate::PartialEqNoBound,
+			#scrate::__private::codec::Encode,
+			#scrate::__private::codec::Decode,
+			#scrate::__private::scale_info::TypeInfo,
+		)]
+		pub struct SkipCheckIfFeeless<T: #scrate::sp_runtime::traits::SignedExtension>(T);
```

Comment:
any reason why we need to generate this from macro?

Reply:
Not particularly, happy to move it out.